### PR TITLE
Using angular translate for translation 

### LINKED ISSUE #2: Start translation and localization
We should be using the same method we did on Tiwun Mobile.

### PR DESCRIPTION
**Pull request by Alir3z4**
Fixes #2

Using beloved Angular Translate package: https://angular-translate.github.io/docs/#/guide/

The sidebar:
image

First page header:

image

The RTL should be added as well.
However, for now we have no hurries since the language pack is not ready.

### CODE REVIEW

File: app/header/header_controller.js [hunk 1]
```diff
@@ -99,15 +104,19 @@ function HeaderController($scope, $rootScope, $state, $stateParams, $timeout,
       function(data) {
         // Prevent older cache
         entry.cache(true);
-        toaster.success("Done", "Entry created!");
+        $translate(["DONE", "ENTRY_CREATED"]).then(function(translations) {
+          toaster.success(translations.DONE, translations.ENTRY_CREATED);
```
Comment (AmirSavand) [inline app/header/header_controller.js]: hmm

**Comment by Alir3z4** [comment]
For now, I have only translated the header controller and its html view.
Please follow the same path and translate other views.
Any new code should have translations.

File: app/header/header_controller.js (hunk 1 above)
Comment (Alir3z4) [reply app/header/header_controller.js]: This is by this https://angular-translate.github.io/docs/#/guide/03_using-translate-service

In the controller the translation loading is asynchronous

**Comment by Alir3z4** [comment]
let me make this shine now

**Comment by Alir3z4** [comment]
Closing in favor of https://github.com/SavandBros/gonevis-dash/pull/878

**Pull request closed without merging**